source/includes/steps-connect-excel-macos.yaml

```diff
@@ -108,4 +108,9 @@ content: |
       .. figure:: /images/bi-connector/excel-import-data-final-example-mac.png
          :alt: Screenshot of imported data in a new sheet
          :figwidth: 700px
+
+  .. note::
+
+            Special characters may not display correctly on macOS due
+            to a known issue.
```

Comment:
Just a thought - I wonder if we should specify that this issue is with Excel, and not the BIC?

Reply:
Added the example of letters with accent marks.  Fixed.

Re: attributing fault.  I pinged Patrick Meredith about this.  We're not 100% certain that the fault is with Excel, so it's best not to document something like that.  Microsoft could sue us if it turns out they are not at fault.

Reply:
Yes, I think it is best not to attribute fault, even though I am 99% certain it is a problem with Excel for Mac, which seems to be calling ANSI routines even when given a Unicode driver, something Unicode drivers support for backward compatibility.